features/dropdowns/relevance_default.feature

```diff
@@ -0,0 +1,9 @@
+  Feature: Default sort is relevance
+    In order to identify the most relevant results
+    As a researcher
+    I wan the default sort option to be Relevance
+
+  Scenario: Default sort option is Relevance
+    Given I am on the default search page
+    When I search on the phrase "bloch"
+    Then "Sort by relevance" should be selected for "sort" dropdown
```

Comment:
@barnabyalter Thanks for your feedback yesterday. This definitely makes sense to me. I'll move this up a level and call it sorting.feature, and start from "Given I am on the brief results screen".

I'm assuming I should actually have 2 here, though:
Given I am on the brief results screen
Then the results should be sorted by relevance

and 

Given I am on the brief results screen
When I select "relevance" from the "Sort by" dropdown 
Then the results should be sorted by relevance

This covers the default versus selected. And you think the "results should be sorted by relevance" should not actually look at the sorting, but rather just at whether the correct option is selected? 

An alternate approach would be to use a more broad search, like "archives", that returns more than one hit and confirm which is first for different options.

In any case, I'll post a few more commits to the PR later this afternoon, then move on to other stories tomorrow.